ui/admin/app/routes/scopes/scope/sessions/index.js

```diff
@@ -169,12 +182,27 @@ export default class ScopesScopeSessionsIndexRoute extends Route {
    */
   @action
   async refreshAll() {
-    const { id: scope_id } = this.modelFor('scopes.scope');
-
-    await this.getAllSessions(scope_id);
-    await this.getAssociatedUsers();
-    await this.getAssociatedTargets();
+    const scope = this.modelFor('scopes.scope');
+    const orgScope = await this.store.findRecord('scope', scope.scope.id);
+    const globalScope = await this.store.findRecord('scope', 'global');
+
+    await this.getAllSessions(scope.id);
+    if (
+      this.can.can('list model', globalScope, { collection: 'users' }) &&
+      this.can.can('list model', orgScope, { collection: 'users' })
+    ) {
+      await this.getAssociatedUsers();
+    }
+    if (this.can.can('list model', scope, { collection: 'targets' })) {
+      await this.getAssociatedTargets();
+    }
 
     return super.refresh(...arguments);
   }
+
+  setupController(controller) {
+    const scope = this.modelFor('scopes.scope');
+    super.setupController(...arguments);
+    controller.setProperties({ scope });
```

Comment:
Is this used in the sessions template?

Reply:
Nope. Thanks for catching this. I thought I needed it in the template but ended up not and didn't remove this.